examples/config-generator/assets/couette.xml.template

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0"?>
+<scenario-configuration>
+  <couette-test>
+    <domain
+      channelheight="{cfd-size}"
+      wall-init-velocity="0.0 ; 0.0 ; 0.0"
+      wall-init-cycles="0"
+      wall-velocity="{wall-velocity} ; 0.0 ; 0.0"
+      wall-oscillations="0"
+    />
+    <coupling
+      coupling-cycles="{coupling-cycles}"
+      two-way-coupling="{two-way-coupling}"
+      send-from-md-to-macro="yes"
+      filter-init-cycles="0"
+      write-csv-every-timestep="10"
+      send-from-macro-to-md="yes"
+      compute-snr="no"
+    />
+    <macroscopic-solver
+      type="{cfd-solver}"
+      viscosity="2.14"
+      number-of-processes="1 ; 1 ; 1"
+      plot-every-timestep="0"
+      init-advance-cycles="0"
+      foam-setup-directory="path/to/FoamSetup/parent/foler"
+      foam-setup-folder="FoamSetup"
+      boundaries-with-MD="0; 0; 0; 0; 0; 0; 1; 1; 1; 1; 1; 1"
+    />
+    <microscopic-solver
+      type="{md-solver}"
+      temperature="1.1"
+      equilibration-steps="{equilibration-steps}"
+      number-md-simulations="{num-md-sims}"
+      density="0.813037037"
+    />
+  </couette-test>
+
+  <mamico>
+    <coupling-cell-configuration
+      cell-size="{cell-size} ; {cell-size} ; {cell-size}"
+      linked-cells-per-coupling-cell="{linked-cells-per-coupling-cell} ; {linked-cells-per-coupling-cell} ; {linked-cells-per-coupling-cell}"
+      write-every-microscopic-timestep="0"
+      microscopic-filename="CouplingCell_micro"
+      write-every-macroscopic-timestep="0"
+      macroscopic-filename="CouplingCell_macro"
+    />
+    <momentum-insertion type="nie-velocity-imposition" outermost-overlap-layer="2" innermost-overlap-layer="3" />
+    <transfer-strategy type="nie-transfer" mass-flux-west="yes" mass-flux-east="yes" mass-flux-north="no" mass-flux-south="no" mass-flux-bottom="no" mass-flux-top="no" shift-by-timesteps="0.5"/>
+    <parallel-topology type="xyz" />
+    <thermostat type='outerLayers' number-layers='1' />
+    {boundary-condition-xml}
+  </mamico>
+
+  <molecular-dynamics>
+    <molecule-configuration
+      mass="1.0"
+      temperature="1.27"
```

Comment:
ls1 does not need the temperature to start higher and then go lower, so this can be 1.1, however ls1 also ignores this, and also this is a default test run, so maybe it's fine like this

Reply:
I think if ls1 ignores it, it should be OK. This sort of stuff should not be documented in the (generated) config files, IMHO, but in the wiki instead. (E.g. "**Only SimpleMD:** [the explanation]")
Should the temperature also be 1.1 for SimpleMD?